modules/serverless-eventing/pages/service-mesh/eventing-service-mesh-mt-channel-based-broker-authorization.adoc

```diff
@@ -0,0 +1,328 @@
+----
++
+.Example output
+[source,terminal]
+----
+2023/06/13 10:17:04 sending cloudevent to http://broker-ingress.knative-eventing.svc.cluster.local/authz-tests/channel-based-broker-br
+2023/06/13 10:17:04 failed to send cloudevent: 403:
+2023/06/13 10:17:05 sending cloudevent to http://broker-ingress.knative-eventing.svc.cluster.local/authz-tests/channel-based-broker-br
+2023/06/13 10:17:05 failed to send cloudevent: 403:
+----
+
+.Authorize source to post events to Knative Broker
+
+In the previous section, we denied access to Knative Eventing workloads, to grant permissions to
+post events to a Knative Broker with class MTChannelBasedBroker.
+
+. Create a `AuthorizationPolicy` in the `knative-eventing` namespace to allow pods
+in the `authz-tests` namespace to send events to Knative Brokers in the same `authz-tests` namespace:
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-authz-tests-mt-channel-based-broker
+  namespace: knative-eventing
+spec:
+  action: ALLOW
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: "broker-ingress" <2>
+  rules:
+    - from: <1>
+        - source:
+            namespaces: [ "authz-tests" ]
+      to: <2>
+        - operation:
+            methods: [ "POST" ]
+            paths: [ "/authz-tests/*" ] <3>
+----
+<1> Allow workloads in the `authz-tests` namespace
+<2> To post events to Knative brokers in the `authz-tests` namespace.
+<3> Knative Broker with class MTChannelBasedBroker accepts events on HTTP path following the pattern: `/<broker-namespace>/<broker-name>`.
+
+. Apply the `AuthorizationPolicy` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Create a `AuthorizationPolicy` in the `knative-eventing` namespace to allow `mt-broker-ingress` in
+the `knative-eventing` namespace to post events to the `InMemoryChannel` dispatcher:
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-mt-channel-based-broker-ingress-to-imc-dispatcher
+  namespace: knative-eventing
+spec:
+  action: ALLOW
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: "imc-dispatcher" <2>
+  rules:
+    - from: <1>
+        - source:
+            namespaces: [ "knative-eventing" ]
+            principals: [ "cluster.local/ns/knative-eventing/sa/mt-broker-ingress" ]
+      to: <2>
+        - operation:
+            methods: [ "POST" ]
+----
+<1> Allow `mt-broker-ingress` in the `knative-eventing` namespace.
+<2> To post events to the `InMemoryChannel` dispatcher.
+
+. Apply the `AuthorizationPolicy` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Create a `AuthorizationPolicy` in the `knative-eventing` namespace to allow `imc-dispatcher` in
+the `knative-eventing` namespace to post events to the `mt-broker-filter`:
```

Comment:
I guess same as above. but I am not really sure if this doc needs to explain all in depth :-)